docker-compose.yml

```diff
@@ -175,7 +175,11 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock:ro
       - ./vector/vector.yaml:/etc/vector/vector.yaml:ro
     command:
-      ['--config', '/etc/vector/vector.yaml', '--require-healthy', 'true']
+      - '--config'
+      - '/etc/vector/vector.yaml'
+      - '--require-healthy'
+      - 'true'
+      - '--watch-config'
```

Comment:
(I'm also breaking out this array into a vertical YAML format, just because the arguments list is getting kind of long and this will make it somewhat easier to add other arguments in the future, if needed.)